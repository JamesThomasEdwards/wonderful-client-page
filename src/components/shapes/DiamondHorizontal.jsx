// React;
import React from 'react';
// Props;
import PropTypes from 'prop-types';

const propTypes = {
    styleColor: PropTypes.string,
    styleImage: PropTypes.string,
};

const DiamondHorizontal = (props) => {
    return (
        <div className={props.styleColor ? props.styleColor : "diamond-horizontal"}>
            <div className={props.styleImage}>
            </div>
        </div>
    );
};

DiamondHorizontal.propTypes = propTypes;

export default DiamondHorizontal;
