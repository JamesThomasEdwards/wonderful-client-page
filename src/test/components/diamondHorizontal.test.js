// React;
import React from 'react';
// Files;
import DiamondHorizontal from '../../components/shapes/DiamondHorizontal.jsx';
// Testing
import ReactShallowRenderer from 'react-test-renderer/shallow';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

test('should render App component correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<DiamondHorizontal />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});