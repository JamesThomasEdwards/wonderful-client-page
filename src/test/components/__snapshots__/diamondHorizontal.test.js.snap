// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component correctly 1`] = `
<div
  className="diamond-horizontal"
>
  <div />
</div>
`;
